File: Baekjoon/python/5단계/다이얼.py
s = list(input())
alphabets = ['ABC', 'DEF', 'GHI', 'JKL', 'MNO', 'PQRS', 'TUV', 'WXYZ']
result = 0
for letter in s:
    for i in range(len(alphabets)):
        if letter in alphabets[i]:
            result += i + 3
print(result)
    

File: Baekjoon/java/5단계/다이얼.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] numbers = {"ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"};
        String[] input = br.readLine().split("");
        int result = 0;

        for (String letter : input) {
            for(int i = 0; i < numbers.length; i++){
                String number = numbers[i];
                if(number.contains(letter)){
                    result += i + 3;
                }
            }
        }

        System.out.print(result);
    }
}